perun-core/src/main/java/cz/metacentrum/perun/core/api/AuditMessagesManager.java

```diff
@@ -62,13 +62,14 @@ public interface AuditMessagesManager {
 	List<AuditMessage> getMessagesByIdAndCount(PerunSession perunSession, int id, int count) throws PrivilegeException;
 
 	/**
-	 * Returns page of audit messages. Query parameter specifies offset and page size. Total count is only estimated.
+	 * Returns page of audit messages filtered by name of event. Query parameter specifies offset and page size. Total count is only estimated.
 	 *
 	 * @param perunSession perun session
+	 * @param eventName name of event
 	 * @return Page of audit messages
 	 * @throws InternalErrorException When implementation fails
 	 */
-	Paginated<AuditMessage> getMessagesPage(PerunSession perunSession, MessagesPageQuery query) throws PrivilegeException;
+	Paginated<AuditMessage> getMessagesPage(PerunSession perunSession, MessagesPageQuery query, String eventName) throws PrivilegeException;
```

Comment:
instead of putting new parameters to paging methods, we put the parameters into the PageQuery object  (see `UsersPageQuery`) and then retrieve the parameters from there